### PR TITLE
✨ Make early access project scope

### LINKED ISSUE #324: 🚸 Make early acess status project scope instaed of global scope
#### Requested Feature
Currently, the early access status is set globally so when users user set early access, any other project they make will have the early access status. This causes confusion and the early access status should only be limited to the project scope.

### PR DESCRIPTION
**Pull request by 12944qwerty**
#### Summary
Changes the early access behavior to project scope.
Specifying early access when creating one project does not enable early access globally. Must use `pros --use-early-access=True` to enable early access by default.
Even if early access is enabled by default, you can use PROS 3 with `pros c n <name> --no-early-access` for that project.

Closes #324

### CODE REVIEW

File: pros/cli/conductor.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def fetch(query: c.BaseTemplate):
               help="Force apply the template, disregarding if the template is already installed.")
 @click.option('--remove-empty-dirs/--no-remove-empty-dirs', 'remove_empty_directories', is_flag=True, default=True,
               help='Remove empty directories when removing files')
-@click.option('--early-access/--disable-early-access', '--early/--disable-early', '-ea/-dea', 'early_access', '--beta/--disable-beta', default=None,
+@click.option('--early-access/--no-early-access', '--early/--no-early', '-ea/-nea', 'early_access', '--beta/--no-beta', default=None,
```
Comment (ayushuk) [inline pros/cli/conductor.py:97]: Imo "disable" > "no". Was there a reason we wanted to change this?

File: pros/cli/conductor.py [hunk 2]
```diff
@@ -258,8 +258,8 @@ def new_project(ctx: click.Context, path: str, target: str, version: str,
               help='Force update all remote depots, ignoring automatic update checks')
 @click.option('--limit', type=int, default=15,
               help='The maximum number of displayed results for each library')
-@click.option('--early-access/--disable-early-access', '--early/--disable-early', '-ea/-dea', 'early_access', '--beta/--disable-beta', default=None,
-              help='View a list of early access templates')
+@click.option('--early-access/--no-early-access', '--early/--no-early', '-ea/-nea', 'early_access', '--beta/--no-beta', default=None,
+              help='Create a project using the PROS 4 kernel')
```
Comment (ayushuk) [inline pros/cli/conductor.py]: This is for `query_templates` we should change the description back since `query_templates` doesn't create a new project.

**Review by ayushuk — CHANGES_REQUESTED** [review]
Overall, I think this is a good implementation. I'm thinking what if we added a QOL feature where if you are already in a early access project, `query-templates` returns early access templates. At the moment you have to specify the -ea flag to see them. It isn't a big deal since `query-templates` is already a command you can run globally but it could create confusion since users would expect the templates in respect to the context of the project. The `resolve_templates` doesn't already have project info so we would just need to figure out if there's a project.pros in the current directory and read that. This could be something for later or for this PR if you have the chance. Address the comments and we can merge this.

File: pros/cli/conductor.py (hunk 1 above)
Comment (12944qwerty) [reply pros/cli/conductor.py:97]: Disable makes it sound like we're disabling it globally instead of just for that project.

**Review by ayushuk — APPROVED** [review]
LGTM

**Pull request merged**